src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -30,7 +30,6 @@
 		010A47562754C1BC000AA80E /* VersionError.swift in Sources */ = {isa = PBXBuildFile; fileRef = 010A47552754C1BC000AA80E /* VersionError.swift */; };
 		010A47662754C74F000AA80E /* TicketValidationResultTokenResource.swift in Sources */ = {isa = PBXBuildFile; fileRef = 010A47652754C74F000AA80E /* TicketValidationResultTokenResource.swift */; };
 		010A47682754C7E7000AA80E /* TicketValidationResultTokenSendModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 010A47672754C7E7000AA80E /* TicketValidationResultTokenSendModel.swift */; };
-		010A476A2754D5F6000AA80E /* TicketValidationResultTokenLocator.swift in Sources */ = {isa = PBXBuildFile; fileRef = 010A47692754D5F6000AA80E /* TicketValidationResultTokenLocator.swift */; };
```

Comment:
The comment is not regarding this line, but in general, I think the folders HTTPClient/Resources and HTTPClient/Service/LocatorsNotYetUsed and parts of HTTPClient/Security shouldn't be part of the HTTPClient folder. These files are containing business data and are not belonging to the core component implementation.

Reply:
So you propose to move the resources from `HTTPClient/Resources` to the location where they are used?
And what about the `LocatorsNotYetUsed`? Where to move them?

What about renaming `HTTPClient` to `HTTPClientCore` and then creating on the same level a folder called `HTTPClientResources`?

So at the end we can create a own package for the `HTTPClientCore`....

Reply:
Okay an own package is not as easy because we have too many dependencies. This would better be an separate ticket because it takes too much time.
But what I did is rename the `HTTPClient` folder to `HTTPClientCore` and create a new folder `HTTPClient` where all the custom resources and the LocatorsNotYetUsed are contained in.
The security stays in `HTTPClientCore`, because it is needed for the functionality of the `RestServiceProvider` itself.